src/assets/css/Home.scss

```diff
@@ -0,0 +1,13 @@
+.articles{
```

Comment:
please, use indentation of two spaces in all your files.

Reply:
on it

Reply:
resolved